docs/admin/resourcequota/index.md

```diff
@@ -57,11 +54,12 @@ in a namespace can be limited.  The following compute resource types are support
 
 | ResourceName | Description |
 | ------------ | ----------- |
-| cpu | Total cpu requests of containers |
-| memory | Total memory requests of containers
-
-For example, `cpu` quota sums up the `resources.requests.cpu` fields of every
-container of every pod in the namespace, and enforces a maximum on that sum.
+| cpu | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
```

Comment:
Let's have all the Resource Names be surrounded in `backticks` (and `ResourceName` itself if that is the name of an object)